app/views/stations/show.html.erb

```diff
@@ -1,10 +1,40 @@
 <%= @station.name %>
 <%= @station.address %>
+
+<% if !@review.nil? %>
+<% if @review.errors.any? %>
+<% @review.errors.full_messages.each do |error| %>
+<%= error %>
+<% end %>
+<% end %>
+<% end %>
+
+<% if !@reviews.nil? %>
 <ul>
-  <% @station.reviews.each do |review| %>
-    <li>
-      <%= review.description %>
-      <%= review.rating %>
-    </li>
+  <% @reviews.each do |review| %>
+  <li>
+    <%= review.description %>
+    <%= review.rating %>
+    <%= link_to "Edit", edit_station_review_path(@station, review) %>
+  </li>
   <% end %>
 </ul>
+<% end %>
+
+<%= form_for [@station, @review] do |f| %>
+<div>
+  <%= f.label :description %>
+  <%= f.text_area :description %>
+</div>
+
+
+<div>
+  <%= f.label :rating %>
+  <%= f.select(:rating, %w[1 2 3 4], {prompt: 'Rating'}) %>
```

Comment:
We will revisit this for the next section, for now (just to get deployed to heroku and because we are most likely redoing our rating system) we will leave this as is, as it still limits input to desired ratings.